src/cmd/ksh93/bltins/misc.c

```diff
@@ -506,18 +506,15 @@ int	b_universe(int argc, char *argv[],Shbltin_t *context)
 #endif /* cmd_universe */
 
 #if SHOPT_FS_3D
-#if _UWIN
-#include <sys/mount.h>
-#endif
-#   if 0
+#if 0
```

Comment:
When making these types of cleanups try to avoid superfluous cleanups such as removing the whitespace here.

Reply:
I first though it wad a weird case of
```c
          if 0
```
w/o parentheses, then I spotted the '#' at the beginning of the line.
Code is read more often that it is written; so this optical illusion causes one to pause and think about it.
But, no biggie.
Won't change it in future.

Reply:
It's not that it shouldn't be changed. It's that you shouldn't mix functional changes with style changes. Mixing the two makes it considerably harder to review the change and determine if the functional changes are correct.